webconsole/v1/types.go

```diff
@@ -42,6 +42,10 @@ type ClusterInfo struct {
 	// LogoutPublicURL is an optional, absolute URL to redirect web browsers to after logging out of the web
 	// console. If not specified, the built-in logout page is shown.
 	LogoutPublicURL string `json:"logoutPublicURL" protobuf:"bytes,5,opt,name=logoutPublicURL"`
+
+	// ClusterResourceOverridesEnabled indicates that the cluster is configured for overcommit and the web console
+	// should hide request and limit fields that will be overridden in its editors
```

Comment:
Should we be more clear in this comment that we assume this means all cluster resource overrides are in use and all memory and cpu fields will be hidden except for the memory limit?

Reply:
Yeah I'll update it with more detail